lib/webtransport.js

```diff
@@ -69,14 +70,35 @@ const sessions = new WeakMap()
    * @param {WebTransportCloseInfo} [closeinfo]
    */
   close(closeinfo) {
-    return sessions.get(this).close(closeinfo)
+    const session = sessions.get(this)
```

Comment:
Now tsc knows what type the `WeakMap` is, we need to null-guard on the values, even though they'll be defined.